PendingReleaseNotes

```diff
@@ -12,6 +12,9 @@
   of the column showing the state of a group snapshot in the unformatted CLI
   output is changed from 'STATUS' to 'STATE'. The state of a group snapshot
   that was shown as 'ok' is now shown as 'complete', which is more descriptive.
+* Ceph: MGR's always-on modulues/plugins can now be force-disabled. This can be
```

Comment:
Shouldn't this be MGR specific rather than Ceph in general?

Reply:
Changed.

Reply:
MGR isn't an acronym.  Suggest "Ceph Manager always-on..."